### ISSUE TITLE
Missing dependency kf5-kio-extras -> kf5-kdelibs4support -> kf5-kemoticons

### ISSUE THREAD

**Issue by oprypin** [RJVB/macstrop#11]
```bash
port install kf5-kio-extras
```
```
...
:info:configure CMake Warning at /opt/local/share/cmake-3.8/Modules/CMakeFindDependencyMacro.cmake:55 (find_package):
:info:configure   By not providing "FindKF5Emoticons.cmake" in CMAKE_MODULE_PATH this project
:info:configure   has asked CMake to find a package configuration file provided by
:info:configure   "KF5Emoticons", but CMake did not find one.
:info:configure   Could not find a package configuration file provided by "KF5Emoticons"
:info:configure   (requested version 5.35.0) with any of the following names:
:info:configure     KF5EmoticonsConfig.cmake
:info:configure     kf5emoticons-config.cmake
:info:configure   Add the installation prefix of "KF5Emoticons" to CMAKE_PREFIX_PATH or set
:info:configure   "KF5Emoticons_DIR" to a directory containing one of the above files.  If
:info:configure   "KF5Emoticons" provides a separate development package or SDK, be sure it
:info:configure   has been installed.
:info:configure Call Stack (most recent call first):
:info:configure   /opt/local/lib/cmake/KF5KDELibs4Support/KF5KDELibs4SupportConfig.cmake:40 (find_dependency)
:info:configure   /opt/local/share/ECM/find-modules/FindKF5.cmake:74 (find_package)
:info:configure   CMakeLists.txt:26 (find_package)
:info:configure CMake Warning at /opt/local/share/ECM/find-modules/FindKF5.cmake:74 (find_package):
:info:configure   Found package configuration file:
:info:configure     /opt/local/lib/cmake/KF5KDELibs4Support/KF5KDELibs4SupportConfig.cmake
:info:configure   but it set KF5KDELibs4Support_FOUND to FALSE so package
:info:configure   "KF5KDELibs4Support" is considered to be NOT FOUND.  Reason given by
:info:configure   package:
:info:configure   KF5KDELibs4Support could not be found because dependency KF5Emoticons could
:info:configure   not be found.
:info:configure Call Stack (most recent call first):
:info:configure   CMakeLists.txt:26 (find_package)
:info:configure -- Could NOT find KF5KDELibs4Support (found version "5.35.0"), checked the following files:
:info:configure     /opt/local/lib/cmake/KF5KDELibs4Support/KF5KDELibs4SupportConfig.cmake (version 5.35.0)
:info:configure     Reason given by package: KF5KDELibs4Support could not be found because dependency KF5Emoticons could not be found.
:info:configure CMake Error at /opt/local/share/cmake-3.8/Modules/FindPackageHandleStandardArgs.cmake:137 (message):
:info:configure   Could NOT find KF5 (missing: KDELibs4Support) (found suitable version
:info:configure   "5.35.0", minimum required is "5.3.0")
:info:configure Call Stack (most recent call first):
:info:configure   /opt/local/share/cmake-3.8/Modules/FindPackageHandleStandardArgs.cmake:377 (_FPHSA_FAILURE_MESSAGE)
:info:configure   /opt/local/share/ECM/find-modules/FindKF5.cmake:110 (find_package_handle_standard_args)
:info:configure   CMakeLists.txt:26 (find_package)
:info:configure -- Configuring incomplete, errors occurred!
:info:configure See also "/opt/local/var/macports/build/_Users_oprypin_repos_macstrop_kf5_kf5-kio-extras/kf5-kio-extras/work/build/CMakeFiles/CMakeOutput.log".
...
```

Manually installing *kf5-kemoticons* allowed me to install *kf5-kio-extras*.

**Comment by RJVB** [comment]
Thanks, that's actually a hidden runtime dependency of the KDELibs4Support framework... which is now declared as such.

I missed it because it's completely silent when the KEmoticons framework is installed.

**Comment by RJVB** [comment]
Fixed in 0fe74ee5f00be3f34e815daee979ad40b9eadc48
